HTAN.model.csv

```diff
@@ -68,14 +68,14 @@ Imaging Level 4,Derived imaging data: Object-by-feature array,,"Component, Filen
+RPPA Level 4,"description",,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, Assay Type, Batch Correction Method",,FALSE,Assay,Biospecimen,,
+Mass Spectrometry Level 1,"Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics.",,"Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Data File ID, MS Batch ID, MS-based Assay Type, Analyte Type, MS-based Targeted, Instrument Make and Model, MS Source, Polarity, Mass Range Low Value, Mass Range High Value, Data Collection Mode, MS Scan Mode, MS Labeling, Protocol Link, LC Instrument Vendor and Model, LC Column Vendor and Model, LC Resin, LC Length Value, LC Temp Value, LC ID Value, LC Flow Rate, LC Gradient, LC Mobile Phase A, LC Mobile Phase B, Software and Version, Protocol Link",,FALSE,Assay,,,
 Imaging Level 3 Channels,Channel-level Metadata Attributes,,"HTAN Channel Metadata ID, Channel ID, Channel Name, Channel Passed QC, Cycle Number, Sub Cycle Number, Antibody Role, Target Name, Antibody Name, RRID identifier, Fluorophore, Clone, Lot, Vendor, Catalog Number, Excitation Wavelength, Emission Wavelength, Excitation Bandwidth, Emission Bandwidth, Metal Isotope Element, Metal Isotope Mass, Oligo Barcode Upper Strand, Oligo Barcode Lower Strand, Dilution, Concentration",,FALSE,Assay,,,
 HTAN Channel Metadata ID,HTAN ID for this channel metadata table (same for all rows),,,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
-Channel ID,This must match the corresponding field in the OME-XML / TIFF header. (eg 'Channel:0:1'),,,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
-Channel Name,This must match the corresponding field in the OME-XML / TIFF header. (eg 'Blue' or 'CD45' or “E-cadherin”),,,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
-Channel Passed QC,Identify stains that did not pass QC but are included in the dataset.,"Yes, No - Channel Failed QC",,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
-No - Channel Failed QC,Channel failed QC,,Channel QC Faliure Type,,FALSE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
+Channel ID,"This must match the corresponding field in the OME-XML / TIFF header. (eg 'Channel:0:1')",,,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
+Channel Name,"This must match the corresponding field in the OME-XML / TIFF header. (eg 'Blue' or 'CD45' or 'E-cadherin')",,,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,
+Channel Passed QC,Identify stains that did not pass QC but are included in the dataset.,"Yes, No - Channel Failed QC",,,TRUE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,No - Channel Failed QC,Channel failed QC,,Channel QC Faliure Type,,FALSE,Imaging Level 3 Channels,,https://www.miti-consortium.org/,

```

Comment:
Seem to have lost a line break between `https://www.miti-consortium.org/` and `No - Channel Failed QC`

Reply:
This is currently causing the conversion error `pandas.errors.ParserError: Error tokenizing data. C error: Expected 10 fields in line 102, saw 19`